server/auth/registration.go

```diff
@@ -0,0 +1,42 @@
+package auth
+
+import (
+	"errors"
+
+	"github.com/8tomat8/SSU-Golang-252-Chat/loger"
+	"github.com/jinzhu/gorm"
+)
+
+// UserRegistry is a structure for Messager
+type UserRegistry struct {
+	UserName string `json:"user_name"`
+	Password string `json:"password"`
+	NickName string `json:"nick_name"`
+}
+
+type User struct {
```

Comment:
This structure should be stored in `model` package to be common for all other packages.